client/app/scripts/superdesk-authoring/authoring.js

```diff
@@ -563,9 +564,9 @@
                 !current_item.embargo && current_item.package_type !== 'takes' &&
                 current_item.state !== 'killed' && !current_item.publish_schedule;
 
-            action.create_broadcast = (!_.contains(['spiked', 'scheduled', 'killed'], current_item.state)) &&
-                (_.contains(['published', 'corrected'], current_item.state)) &&
-                current_item.type === 'text' && !isBroadcast;
+            action.create_broadcast = (_.contains(['published', 'corrected'], current_item.state)) &&
+                _.contains(['text', 'preformatted'], current_item.type) &&
```

Comment:
its not covered. `isBroadcast` is set to `True` for broadcast item and false for the rest. we need this so that `create_broadcast` action available for text items (which are not broadcast item) and not for image, audio and video.